app/uk/gov/hmrc/timetopayproxy/models/CreatePlanResponse.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.timetopayproxy.models
+
+import play.api.libs.json.Json
+
+final case class CreatePlanResponse(customerReference: String, pegaPlanId: String, planStatus: String)
```

Comment:
Might be better to use CustomerReference value typee instead of String.

Reply:
whoops i missed using the value class 🙄